object-manipulation-1/src/get-value.ts

```diff
@@ -0,0 +1,8 @@
+/* exported getValue */
+function getValue(object: Record<string, any>, key: string): any {
+  if (key in object) {
+    return object[key];
```

Comment:
Try to use `unknown` in the future instead of `any`. `unknown` still has a form of type checking whereas `any` completely ignores all type checking